packages/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -83,22 +97,33 @@ const Breadcrumbs: FC<BreadcrumbsProps> = ({
             hiddenBreadcrumbs.map((breadcrumb, i) => {
```

Comment:
Is it possible to keep breadcrumbs in the DOM and hide it with css? If dom is not available in server side rendered html, it might cause crawling issues.

cc @Nazzanuk